new-lamassu-admin/src/components/booleanPropertiesTable/booleanPropertiesTable.js

```diff
@@ -0,0 +1,113 @@
+import { makeStyles } from '@material-ui/core/styles'
+import React, { useState, memo } from 'react'
+
+import { H4 } from 'src/components/typography'
+import { Link } from 'src/components/buttons'
+import { RadioGroup } from 'src/components/inputs'
+import { Table, TableBody, TableRow, TableCell } from 'src/components/table'
+import { startCase } from 'src/utils/string'
+import { ReactComponent as EditIcon } from 'src/styling/icons/action/edit/enabled.svg'
+import { ReactComponent as EditIconDisabled } from 'src/styling/icons/action/edit/disabled.svg'
+import { ReactComponent as TrueIcon } from 'src/styling/icons/table/true.svg'
+import { ReactComponent as FalseIcon } from 'src/styling/icons/table/false.svg'
+
+import { booleanPropertiesTableStyles } from './booleanPropertiesTable.styles'
+
+const BooleanPropertiesTable = memo(
+  ({ title, disabled, data, elements, save }) => {
+    const [editing, setEditing] = useState(false)
+    const [radioGroupValues, setRadioGroupValues] = useState(elements)
+
+    const innerSave = () => {
+      radioGroupValues.forEach(element => {
+        data[element.name] = element.value
+      })
+
+      save(data)
+      setEditing(false)
+    }
+
+    const innerCancel = () => {
+      setEditing(false)
+    }
+
+    const handleRadioButtons = (elementName, newValue) => {
+      setRadioGroupValues(
+        radioGroupValues.map(element =>
+          element.name === elementName
+            ? { ...element, value: newValue }
+            : element
+        )
+      )
+    }
+
+    const radioButtonOptions = [
+      { label: 'Yes', value: true },
+      { label: 'No', value: false }
+    ]
+
+    if (!elements || radioGroupValues?.length === 0) return null
+
+    const useStyles = makeStyles(booleanPropertiesTableStyles)
+
+    const classes = useStyles()
```

Comment:
Make styles should be outside of the component and hooks calls should be the first thing on the component.

Reply:
Roger! But why exactly hooks should be the first thing?

Reply:
I also couldn't think of a better name for this component... any suggestions?